5.2/5.2.4/debian/docker-initialize.py

```diff
@@ -224,6 +224,8 @@ def buildout(self):
         versions = self.env.get("PLONE_VERSIONS",
                    self.env.get("VERSIONS", "")).strip().split()
 
+        stdlogging = self.env.get("STD_LOGGING")
```

Comment:
Should go to default behavior.

Reply:
Hi @avoinea, sorry for the delay. I changed the default with  stdout/stderr logging. So the command to switch to legacy behavior might be:
`docker run -p8080:8080 -e FILE_LOGGING=1 plone:5.2,4-debian
`
If you agree, I could spread the changes to the others 5.2.4 images (alpine, ...). 
I think I leave the other versions of Plone as they are.

Reply:
Yes. Usually we don't rebuild existing tags, thus this will go into the next releases.